docs/content/config/advanced/auth-ldap.md

```diff
@@ -6,37 +6,188 @@ title: 'Advanced | LDAP Authentication'
 
-!!! example "saslauthd"
+???+ example "Example Code: A really simple `LDAP_QUERY_FILTER` configuration, using only the user filter and allowing only `admin@*` to spoof any sender addresses"
 
-    - `SASLAUTHD_LDAP_FILTER`
+    ```yaml
+    - ENABLE_LDAP=1
+    - LDAP_SERVER_HOST=ldap.example.org
+    - LDAP_SEARCH_BASE=dc=example,dc=org"
+    - LDAP_BIND_DN=cn=admin,dc=example,dc=org
+    - LDAP_BIND_PW=mypassword
+    - SPOOF_PROTECTION=1
+    
+    - LDAP_QUERY_FILTER_DOMAIN=(mail=*@%s)
+    - LDAP_QUERY_FILTER_USER=(mail=%s)
+    - LDAP_QUERY_FILTER_ALIAS=(|) # doesn't match anything
+    - LDAP_QUERY_FILTER_GROUP=(|) # doesn't match anything
+    - LDAP_QUERY_FILTER_SENDERS=(|(mail=%s)(mail=admin@*))
+    ```
+
+### `DOVECOT_*_FILTER`
+Those variables specify the LDAP filters that dovecot uses to determine if a user can log in to their IMAP account, and which mailbox is responsible to receive email for a specific postfix user. This is split into two lookups, buth using e.g. `%u` as the placeholder for the full login name ([see dovecot documentation for a full list of placeholders](https://doc.dovecot.org/configuration_manual/config_file/config_variables/)):
+
+- `DOVECOT_USER_FILTER` is used to get the account details (uid, gid, home directory, quota, ...) of a user.
+- `DOVECOT_PASS_FILTER` is used to get the password information of the user as well as to determine if they're allowed to log in at all. TODO: when is this neccessary?
+
+!!! note
+
+    In many cases, both can be the same, in which case you only need to specify `DOVECOT_USER_FILTER`.
+
+If your directory doesn't have the [postfix-book schema](https://github.com/variablenix/ldap-mail-schema/blob/master/postfix-book.schema) installed, then you must change the internal attribute handling for dovecot. For this you have to change the `pass_attr` and the `user_attr` mapping, as shown in the example below:
+
+```yaml
+- DOVECOT_PASS_ATTRS=<YOUR_USER_IDENTIFIER_ATTRIBUTE>=user,<YOUR_USER_PASSWORD_ATTRIBUTE>=password
+- DOVECOT_USER_ATTRS=<YOUR_USER_HOME_DIRECTORY_ATTRIBUTE>=home,<YOUR_USER_MAILSTORE_ATTRIBUTE>=mail,<YOUR_USER_MAIL_UID_ATTRIBUTE>=uid,<YOUR_USER_MAIL_GID_ATTRIBUTE>=gid
+```
+
+!!! note
+
+    For `DOVECOT_*_ATTRS`, you can replace `ldapAttr=dovecotAttr` with `=dovecotAttr=%{ldap:ldapAttr}` for more flexibility, like for example `=home=/var/mail/%{ldap:uid}` or just `=uid=5000`.
+
+    A list of dovecot attributes can be found [in the dovecot documentation](https://doc.dovecot.org/configuration_manual/authentication/user_databases_userdb/#authentication-user-database).
+
+???+ example "Example Code: Setup for a directory that has the [qmail-schema](https://github.com/amery/qmail/blob/master/qmail.schema) installed and that uses `uid`"
```

Comment:
```suggestion
???+ example

    Setup for a directory that has the [qmail-schema](https://github.com/amery/qmail/blob/master/qmail.schema) installed and uses `uid`:
```

Feel free to retain a title here, but I don't think a link should be in it. Omitting a title will fallback to the admonition type, thus with the above suggestion it will be titled "Example" instead.